bindings.c

```diff
@@ -4045,11 +4090,41 @@ static int read_cpuacct_usage_all(char *cg, char *cpuset, struct cpuacct_usage *
 	if (!cpu_usage)
 		return -ENOMEM;
 
+	memset(cpu_usage, 0, sizeof(struct cpuacct_usage) * cpucount);
 	if (!cgfs_get_value("cpuacct", cg, "cpuacct.usage_all", &usage_str)) {
-		rv = -1;
-		goto err;
+		lxcfs_v("failed to read cpuacct.usage_all%s\n", "");
```

Comment:
What's the reason behind the empty string added there? This should compile fine without?

Reply:
Otherwise, the compiler complains:

```
macro.h:7:25: error: expected expression before ')' token
    __func__, __VA_ARGS__);                                \
                         ^
macro.h:10:34: note: in expansion of macro 'lxcfs_debug_stream'
 #define lxcfs_error(format, ...) lxcfs_debug_stream(stderr, format, __VA_ARGS__)
                                  ^
macro.h:19:30: note: in expansion of macro 'lxcfs_error'
 #define lxcfs_v(format, ...) lxcfs_error(format, __VA_ARGS__);
                              ^
bindings.c:4104:3: note: in expansion of macro 'lxcfs_v'
   lxcfs_v("converting cpuacct.usage_percpu into cpuacct.usage_all\n");
   ^
```